dwave/embedding/utils.py

```diff
@@ -18,14 +18,39 @@
 
 import dimod
 import numpy as np
+import math
 
 from six import iteritems
 
 from dwave.embedding.chain_breaks import broken_chains
 
 
-__all__ = 'target_to_source', 'chain_to_quadratic', 'chain_break_frequency', 'adjacency_to_edges'
+__all__ = 'get_chain_strength', 'target_to_source', 'chain_to_quadratic', 'chain_break_frequency', 'adjacency_to_edges'
 
+def get_chain_strength(bqm, prefactor):
```

Comment:
We need a better name for this method. This is not the only way to derive a suggestion for chain strength from a BQM. @alexzucca90, @pau557, any spiffy suggestions?

Reply:
What about `default_clique_chain_strength`? Although it works for sparser problems as well

Reply:
Instead of "default" (which tell you nothing about how it works) I'm hoping we can come up with something more descriptive. That is, if we plan to add more chain strength calculation methods.

Reply:
`scaled_rms_chain_strength()` @alexzucca90 ?

Reply:
Then in the spirit of what @randomir says something like what @hhtong proposes or stuff like `scaled_degree_rms_j_chain_strength`, `spin_glass_transition_chain_strength` or `torque_compensation_chain_strength`.

Reply:
namespaces are our friends, maybe `from dwave.embedding.chain_strengths import scaled_degree_rms`